client/packages/common/src/intl/locales/en/common.json

```
@@ -1278,10 +1278,10 @@
   "messages.confirm-delete-vaccine-courses_other": "This will permanently remove the {{count}} selected vaccine courses",
   "messages.confirm-finalise-rnr": "This will mark the R&R form as finalised and generate an Internal Order.",
   "messages.confirm-inbound-status-as": "Confirm status as $t({{status}})?",
-  "messages.confirm-not-fully-supplied_few": "If you finalise you will not be able to supply any more items to the customer from this requisition. \n\n{{count}} lines in this requisition have not been fully supplied to the customer. \nWould you still like to continue?",
-  "messages.confirm-not-fully-supplied_many": "If you finalise you will not be able to supply any more items to the customer from this requisition. \n\n{{count}} lines in this requisition have not been fully supplied to the customer. \nWould you still like to continue?",
-  "messages.confirm-not-fully-supplied_one": "If you finalise you will not be able to supply any more items to the customer from this requisition. \n\n{{count}} line in this requisition has not been fully supplied to the customer. \nWould you still like to continue?",
-  "messages.confirm-not-fully-supplied_other": "If you finalise you will not be able to supply any more items to the customer from this requisition. \n\n{{count}} lines in this requisition have not been fully supplied to the customer. \nWould you still like to continue?",
+  "messages.confirm-not-fully-supplied_few": "There are {{count}} lines remaining that will no longer be able to be shipped to the customer from this requisition. \n\nWould you still like to continue?",
```

```suggestion
  "messages.confirm-not-fully-supplied_few": "There are {{count}} lines in this requisition that have not been fully supplied to the customer. \n\nWould you still like to continue?",
```

"Shipped" is somewhat reserved keyword for an invoice status 😁

---

Great, good to know and have updated to your suggestion, thank you!